example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -192,7 +221,8 @@ MM_CollectorLanguageInterfaceImpl::scavenger_masterSetupForGC(MM_EnvironmentBase
 void
 MM_CollectorLanguageInterfaceImpl::scavenger_workerSetupForGC_clearEnvironmentLangStats(MM_EnvironmentBase *env)
 {
-	/* Do nothing for now */
+	MM_EnvironmentStandard *envStd = (MM_EnvironmentStandard *)env;
+	MM_SublistFragment::flush((J9VMGC_SublistFragment*)&envStd->_scavengerRememberedSet);
```

Comment:
GC worker threads should not have to explicitly flush their remembered set fragments at the beginning of the GC. Objects get added to an OMR_VMThread's remembered set fragment during application execution or during a GC. If an OMR_VMThread adds an object to it's remembered set fragment during application execution this should be flushed at the beginning of the GC via the API I mentioned. If it happens during a GC it needs to be flushed before the GC finishes.

Sure lets discuss this tomorrow morning but please add a comment here afterwards to make sure we document the results of the conversation for others.